utils/constants.ts

```diff
@@ -1,3 +1,5 @@
 import { getAddress } from "@ethersproject/address";
+import { ethers, constants, BigNumber } from "ethers";
 
 export const ethAddress = getAddress(`0x${"e".repeat(40)}`);
+export const feePercent = BigNumber.from("15").div("100");
```

Comment:
this fee can change and depends on network, but for this PR I'd say this is sufficient! 👍

Reply:
Cool, I will add a TODO and create a ticket so this isn't lost

Reply:
#61

Reply:
thanks, i will close #50 in favor of #61, because it's worded better :+1: